nautobot/dcim/api/serializers.py

```diff
@@ -955,7 +955,7 @@ class Meta:
         ]
 
 
-# TODO: collapse this with InterfaceSerializer in 2.0.
+# 2.0 TODO: collapse this with InterfaceSerializer in 2.0.
```

Comment:
If I understand #2530 correctly, this serializer would become non-default in 2.0, but wouldn't actually be removed until 2.2?